src/chat_commands.c

```diff
@@ -226,13 +226,18 @@ void cmd_group_accept(WINDOW *window, ToxWindow *self, Tox *tox, int argc, char
     }
 
     const char *passwd = NULL;
-    uint16_t passwd_len = 0;
+    size_t passwd_len = 0;
```

Comment:
size_t is a downgrade. I'd probably write a strlen16 function or something that calls strlen and then maxes at UINT16_MAX, or maybe just cast, or if you want to be really fancy, have an error code.

Reply:
I'll just cast it for now. `strlen()` is used ~100 times in toxic so a better solution will need to be its own PR.